package.json

```diff
@@ -1,70 +1,69 @@
 {
-  "name": "speccy",
```

Comment:
Bah, sorry. Indentation was inconsistent and broken, and my editor decided to "fix it".

Only lines changed were:

```
-    "test": "mocha",
-    "cover": "istanbul cover _mocha",
-    "coveralls": "npm run cover -- --report lcovonly -- -R spec && cat ./coverage/lcov.info | coveralls"
+        "test": "nyc --reporter=html --reporter=text mocha",
+        "coverage": "nyc report --reporter=text-lcov | coveralls"
```

Also swapped out the old "istanbul" for "nyc".